### PR TITLE
revamp the README

### LINKED ISSUE #18: Add annovar cmd (non-cavatica)
Add instructions on how to run annovar with correct flags in `README.md` to get correct vcf coordinates for downstream analyses.

### LINKED ISSUE #19: move tool readme to root of repo
instead of within the folder

### PR DESCRIPTION
**Pull request by jharenza**
This PR cleans up the README and addresses the following issues.

Closes #18 
Closes #19 

Please let me know if we should add anything else immediately.

### CODE REVIEW

**Comment by naqvia** [comment]
Looks good to me but one minor change. This is not correct
`Optional: VCF annotation with ClinVar. If not done by the user, ClinVar file is required input to AutoGVP.`

The user can input their own clinVar database file and just feed the file path, but it’s not within the VCF file they input. For the CAVATICA workflow it is by default but not in this "custom" version.

**Comment by naqvia** [comment]
Looks good to me except I think the `clinVar` input info for the custom workflow is unclear. The ClinVar VCF file is not optional... it is required. The database download script downloads and stores the latest version that is then used, but the user can download and input any version they'd like and give it to the tool. The tool then uses that clinVar file to annotate Stars.

**Comment by jharenza** [comment]
> Looks good to me except I think the `clinVar` input info for the custom workflow is unclear. The ClinVar VCF file is not optional... it is required. The database download script downloads and stores the latest version that is then used, but the user can download and input any version they'd like and give it to the tool. The tool then uses that clinVar file to annotate Stars.

Ah, I meant that the user annotating their germline VCF with ClinVar is optional - actually, is that even an option? Or do they always need to provide a VCF

File: README.md [hunk 1]
```diff
@@ -1,78 +1,157 @@
...
+```r
+Rscript 01-annotate_variants_custom_input.R --vcf <*.vcf> --multianno <*multianno.txt> --intervar <*hg38_multianno.txt.intervar> --autopvs1 <*autopvs1.txt --output <output_prefex> --submission input/variant_summary.txt --submission_summary input/submission_summary.txt
+```
+
+### Custom (non-CAVATICA) input ###
+1. Annotate the germline VCF with VEP.
+2. Run ANNOVAR with InterVar with the following options (to create the VCF input for AutoGVP).
+```perl
+perl table_annovar.pl input/test_hg38_selected_VEP_annotated.vcf hg38 --buildver hg38 --out test_hg38_selected --remove --protocol gnomad211_exome,gnomad211_genome --operation f,f --vcfinput
+```
```
Comment (jungkim2) [inline README.md:56]: As above is for ANNOVAR annotation for gnomAD frequency and generating chrom,pos,ref,alt id to be merged with InterVar, we need another step where annotating file with InterVar

File: README.md [hunk 2]
```diff
@@ -1,78 +1,157 @@
...
+3. Run AutoPVS1.
+4. Optional: VCF annotation with ClinVar. If not done by the user, ClinVar file is required input to AutoGVP.
```
Comment (jungkim2) [inline README.md:58]: I have never done this way, so unsure, but based on Ammar's comment, not optional.

File: README.md [hunk 3]
```diff
@@ -1,78 +1,157 @@
...
+```bash
+bash download_db_files.sh
+```
+```r
+Rscript 01-annotate_variants_custom_input.R --vcf input/testing_010423_VEP.vcf --multianno input/testing_010423.hg38_multianno.txt --intervar input/testing_010423_intervar.hg38_multianno.txt.intervar --autopvs1 input/testing_010423_autopvs1.txt --clinvar input/clinvar_20211225.vcf.gz --output SRRT0182 --submission input/variant_summary.txt --submission_summary input/submission_summary.txt
+```
+
+### Step by step workflow
+1. Read in ClinVar file
+2. Create `vcf_id` (chr and position)
```
Comment (jungkim2) [inline README.md:81]: I thought vcf_id was chr_pos_ref_alt (not just chr and position)

File: README.md (hunk 1 above)
Comment (jharenza) [reply README.md:56]: Can you provide the command?

File: README.md (hunk 2 above)
Comment (jharenza) [reply README.md:58]: OK, I need to rephrase this, we do it with the CAVATICA workflow, but user input, it is separate. I will update the figure also.

File: README.md [hunk 4]
```diff
@@ -1,78 +1,157 @@
...
+2. Run ANNOVAR with InterVar with the following options (to create the VCF input for AutoGVP).
+```perl
+perl table_annovar.pl input/test_hg38_selected_VEP_annotated.vcf hg38 --buildver hg38 --out test_hg38_selected --remove --protocol gnomad211_exome,gnomad211_genome --operation f,f --vcfinput
+```
+3. Run AutoPVS1.
+4. Optional: VCF annotation with ClinVar. If not done by the user, ClinVar file is required input to AutoGVP.
+4. Run AutoGVP.
+
+AutoGVP Requirements (recommended to place all in the `input/` folder):
+- VEP- and ANNOVAR-annotated VCF (`*VEP.vcf`)
```
[the remark this answers is not in the document]
Comment (jharenza) [reply README.md:62]: 👍

File: README.md [hunk 5]
```diff
@@ -1,78 +1,157 @@
...
+- ANNOVAR file (`*hg38_multianno.txt`)
+- InterVar file (`*intervar.hg38_multianno.txt.intervar`)
+- AutoPVS1 file (`*autopvs1.txt`)
+- ClinVar VCF (`clinvar_yyyymmdd.vcf.gz`; optional)
+- `variant_summary.txt`
+- `submission_summary.txt.gz` (can retrieve with `download_db_files.sh`)
```
[the remark this answers is not in the document]
Comment (jharenza) [reply README.md:68]: yeah good catch

File: README.md [hunk 6]
```diff
@@ -1,78 +1,157 @@
...
+Note: the variant_summary and submission_summary files need to be uncompressed.
+
+Example run:
+```bash
+bash download_db_files.sh
+```
+```r
+Rscript 01-annotate_variants_custom_input.R --vcf input/testing_010423_VEP.vcf --multianno input/testing_010423.hg38_multianno.txt --intervar input/testing_010423_intervar.hg38_multianno.txt.intervar --autopvs1 input/testing_010423_autopvs1.txt --clinvar input/clinvar_20211225.vcf.gz --output SRRT0182 --submission input/variant_summary.txt --submission_summary input/submission_summary.txt
```
[the remark this answers is not in the document]
Comment (jharenza) [reply README.md:76]: >I was wondering for input can --submission be changed to variant_summary rather than --submission?

File: README.md (hunk 3 above)
Comment (jharenza) [reply README.md:81]: (i didn't write this, just copied it over) - you are right:

https://github.com/diskin-lab-chop/pathogenicity-assessment/blob/895f6a5787c4cb7fc292222edf4e3e6761c2415b/AutoGVP/01-annotate_variants_custom_input.R#L96

will update

File: README.md [hunk 7]
```diff
@@ -1,78 +1,157 @@
...
+13. Go through entries and adjust evidence variables above based on criterion*
+14. Report final call based on recalculated evidence variables
+
+#### Annotating Stars
+```
+1 = 'criteria_provided,_single_submitter'
+2 = 'criteria_provided,_multiple_submitters'
+3 = 'reviewed_by_expert_panel'
+4 = 'practice_guideline'
+0/Need resolving = 'criteria_provided,_conflicting_interpretations'
```
[the remark this answers is not in the document]
Comment (jharenza) [reply README.md:101]: i will update

**Comment by jharenza** [comment]
@jungkim2 I think I responded to/addressed all comments, and updated figure 1 here and in the paper, if you want to take another look!

**Comment by jharenza** [comment]
Also closes #11

File: README.md (hunk 1 above)
Comment (jungkim2) [reply README.md:56]: python InterVar.py -b hg38 -i input/test_hg38_selected_VEP_annotated.vcf --input_type=VCF -o test_hg38_selected

**Comment by jungkim2** [comment]
Looks good!

**Comment by jharenza** [comment]
Thanks, if you can approve, I can get it merged!

**Review by jungkim2 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jharenza) [reply README.md:56]: updated

**Pull request merged**